drivers/ata8510/firmware/firmware.hex

```diff
@@ -0,0 +1,1006 @@
+:010000002AD5
```

Comment:
Is this the "RF library"?
Will this driver also work with the ATA8515 that does not accept user firmware?

Reply:
Yes, that was my idea - the basic RF functionality should be here - networking stuff will follow.

As for the ATA8515, from what I've read the driver should only require some minor adjustment - but I don't have one to test, so mine is mostly a guess.